release-notes/6.0/preview/api-diff/preview5/.Net/6.0-preview5_System.Text.Json.Node.md

```diff
@@ -0,0 +1,273 @@
+# System.Text.Json.Node
+
+``` diff
+-namespace System.Text.Json.Node {
```

Comment:
Would it be possible/better for this to show up as a change in namespace name rather than massive deletion / addition of everything in the namespace?

Reply:
i can think of a trivial way to automatically detect this. But i can create an issue in arcade to see if we can do this.
https://github.com/dotnet/arcade/tree/main/src/Microsoft.DotNet.AsmDiff most of the core code base is similar to this (in case u r interested)